nomad/rust-integration-tests.nomad

```diff
@@ -159,6 +168,16 @@ job "rust-integration-tests" {
               destination_name = "web-ui"
               local_bind_port  = 1006
             }
+
+            upstreams {
+              destination_name = "scylla"
+              local_bind_port  = 1007
+            }
+
```

Comment:
Nope! Scylla is not a Consul Connect service. You may instead need to feed in `var.graph_db` to integration tests - I can help with that, but you basically pulumi.export it in the main pulumi project